optaplanner-wb-ui/src/main/java/org/optaplanner/workbench/client/OptaPlannerWorkbenchEntryPoint.java

```diff
@@ -75,6 +81,12 @@ public OptaPlannerWorkbenchEntryPoint(final Caller<AppConfigService> appConfigSe
         this.adminPage = adminPage;
         this.translationService = translationService;
         this.placeManager = placeManager;
+        this.appFormerJsBridge = appFormerJsBridge;
+    }
+
+    @PostConstruct
+    protected void setup() {
+        this.appFormerJsBridge.init("optaplanner-wb/org.optaplanner.workbench.OptaPlannerWorkbench");
```

Comment:
If this is a unique identifier of this particular workbench, turning it into a well-named constant will make it clear.